<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtsqlDemo</class>
 <widget class="QMainWindow" name="qtsqlDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtsqlDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_CreateConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>createConnection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CreateTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>createTable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_insertItem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>insertItem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteItem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>deleteItem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_queryItem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>queryItem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>165</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
